skopt/plots.py

```diff
@@ -230,10 +235,11 @@ def partial_dependence(space, model, i, j=None, sample_points=None,
                 row.append(np.mean(model.predict(rvs_)))
             zi.append(row)
 
-        return xi, yi, zi
+        return xi, yi, np.array(zi).T
```

Comment:
On a more serious note. This used to be a `list of lists` as return type. Now it is an array. I had justified `list of lists` because the points were in the original space not the transformed space. Now I think that was a stupid justification and it should just be a `np.array`.